var/spack/repos/builtin/packages/ape/package.py

```diff
@@ -11,7 +11,7 @@ class Ape(Package):
     Theory framework"""
 
     homepage = "https://www.tddft.org/programs/APE/"
-    url = "http://www.tddft.org/programs/APE/sites/default/files/ape-2.2.1.tar.gz"
+    url = "https://www.tddft.org/programs/APE/sites/default/files/ape-2.2.1.tar.gz"
```

Comment:
These URLs lead to "The Top Online Casinos in Australia" which has nothing to do with "A tool for generating atomic pseudopotentials".
You should probably look at this address to find the sources of Ape: https://gitlab.com/ape/ape